lms/djangoapps/verify_student/views.py

```diff
@@ -594,6 +613,10 @@ def _requirements(self, display_steps, is_active):
             self.WEBCAM_REQ: False,
         }
 
+        # Remove the account activation requirement if disabled via waffle
+        if waffle.switch_is_active(DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH):
+            all_requirements.pop(self.ACCOUNT_ACTIVATION_REQ)
```

Comment:
Do we need this here since you are already accounting for the waffle switch in the `is_active` boolean you are passing to this function?